src/components/UI/Pagination/Pagination.test.tsx

```diff
@@ -1,3 +1,4 @@
+import "@testing-library/jest-dom";
 import { fireEvent, render, screen } from "@testing-library/react";
```

Comment:
i think u can do it via setup in package.json
```
    "setupFilesAfterEnv": [
      "@testing-library/jest-dom/extend-expect"
    ]
```
then in your tsconfig, needa put in `"@types/testing-library__jest-dom"` in `types` array too.

https://jestjs.io/docs/en/configuration#setupfilesafterenv-array

this way no need to keep re-import jest-dom in tests i think